client/src/components/AdminContainer/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Layout, Button, Menu } from 'antd';
+import logo from '../../assets/images/logo.png';
+import './style.css';
+
+const { Header } = Layout;
+
+const AdminContainer = ({ buttonContent, contentComponent }) => {
+  return (
+    <>
+      <Layout>
+        <Header>
+          <div className="header-image">
+            <img src={logo} alt="logo" />
+          </div>
+          <div className="header-btn">
+            {buttonContent !== undefined && (
+              <Button type="primary" danger>
```

Comment:
@MohammedAlghazali If every admin page needs to handle buttons differently, it might make more sense to remove the Header from AdminContainer and into a new component, and instead require all of the admin pages to set up their own Header and pass in custom buttons as a child prop.
Alternatively, you could keep this design and then all of the pages could pass in an `onButtonClicked` prop to handle the clicks themselves.

Reply:
all page has the same header and there is one style for the button, but some page doesn't have a button in the header so I make this condition to make header suitable for all page
**page with button**
![701](https://user-images.githubusercontent.com/55782435/78720065-b24b7400-792d-11ea-82e3-456102c45e4c.png)
**page without button**
![702](https://user-images.githubusercontent.com/55782435/78720082-b7102800-792d-11ea-9eba-400c5fd4f83a.png)
is this good ? or there is a better idea

Reply:
I think that’s fine then, if there will always only be one button or none. Then when it’s time to make the button functional, you could pass an onButtonClicked prop as I mentioned above (so then whichever admin page is using this component will be responsible for the clicking logic.

Reply:
yes we need it in the next time so I will handle it now